CHANGELOG.unreleased.md

```diff
@@ -4,7 +4,10 @@
 
 ### Bug fixes
 
+- [Continuous Replication] Fix VHD size guess for empty files [#4105](https://github.com/vatesfr/xen-orchestra/issues/4105)  (PR [#4107](https://github.com/vatesfr/xen-orchestra/pull/4107))
+
 ### Released packages
 
 - xo-server v5.39.0
 - xo-web v5.39.0
+- vhd-lib v0.7.0
```

Comment:
Always add above as it the order of release and `xo-server` depends on `vhd-lib` so it must be released after.